src/BrowserKitDriver.php

```diff
@@ -807,8 +807,14 @@ private function mergeForms(Form $to, Form $from)
      */
     private function getCrawlerNode(Crawler $crawler)
     {
-        $crawler->rewind();
-        $node = $crawler->current();
+        $node = null;
+
+        if ($crawler instanceof \IteratorAggregate) {
```

Comment:
I would rather check for ``Iterator`` (for the old API) than for ``IteratorAggregate``: you never rely on methods from IteratorAggregate.

and this code should have a comment explaining that the goal is to support Symfony 2.3 where ``getNode`` is not public